### ISSUE TITLE
Embed interactive visualization on NILC website

### ISSUE THREAD

**Issue by jtorrez** [Data4Democracy/immigration-connect#7]
The National Immigration Law Center (NILC) is receiving reports of immigration-related arrests and detentions. They want to map this information in real-time as it happens.

The map could be embedded on a page on their website http://www.nilc.org .  Or it could be a separate lightweight single-purpose website running on a subdomain such as http://incidents.nilc.org

The visualization will need to autoupdate, see issue #6 for more information on the updating and see issue #5 for info on the visualization. 

We can put you in contact with the NILC if you need to discuss the structure of their site and how you can get access to it.

**Comment by jtorrez** [comment]
Waiting on NILC to communicate about embedding on website

**Comment by jtorrez** [comment]
The NILC has put this project on hold for now

**Comment by jtorrez** [comment]
Closing this as this project will not be moving forward.
